webview/src/plots/components/ZoomablePlot.tsx

```diff
@@ -12,12 +13,20 @@ interface ZoomablePlotProps {
   spec: VisualizationSpec
   data?: PlainObject
   id: string
+  size: PlotSize
+}
+
+const TitleLimit = {
+  [PlotSize.LARGE]: 70,
```

Comment:
I have a follow up for the axis labels that will fine tune these values a little bit more.